### PR TITLE
Adding unit tests for the util package

### PR DESCRIPTION
**Pull request by tariq1890**
Added unit tests for the util package

### CODE REVIEW

File: util/strutil/strconv_test.go [hunk 1]
```diff
@@ -36,6 +41,17 @@ var linkTests = []linkTest{
 	},
 }
 
+var sanitizeTests = []sanitizeTest{
```
Comment (brian-brazil) [inline util/strutil/strconv_test.go]: Keep this with the function using it. Usually we're do these inline.

**Comment by tariq1890** [comment]
@brian-brazi Thanks for the review. Done!

File: util/strutil/strconv_test.go [hunk 2]
```diff
@@ -23,6 +23,11 @@ type linkTest struct {
 	expectedTableLink string
 }
 
+type sanitizeTest struct {
```
Comment (brian-brazil) [inline util/strutil/strconv_test.go]: You don't need this now

File: promql/engine_test.go [hunk 3]
```diff
@@ -198,7 +199,7 @@ func TestQueryError(t *testing.T) {
 }
 
 func TestEngineShutdown(t *testing.T) {
-	engine := NewEngine(nil, nil, 10, 10*time.Second)
+	engine := NewEngine(nil, prometheus.DefaultRegisterer, 10, 10*time.Second)
```
Comment (brian-brazil) [inline promql/engine_test.go]: Why did you make this change? This could interfere with other tests.

Comment (tariq1890) [reply promql/engine_test.go]: My bad. I forgot to remove this. Thanks!

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**